explorer/interpreter/builtins.cpp

```diff
@@ -24,6 +24,9 @@ void Builtins::Register(Nonnull<const Declaration*> decl) {
     if (it != builtin_indexes->end()) {
       builtins_[it->second] = interface;
     }
+    if (interface->name() == GetName(Builtin::Eq)) {
+      builtins_[static_cast<int>(Builtin::Eq)] = interface;
+    }
```

Comment:
This should not be necessary any more: the above code should handle `Eq` automatically.